disable parallel build for libjson-c
otherwise build breaks with CONFIG_PKG_DEFAULT_PARALLEL enabled

diff --git a/package/libs/libjson-c/Makefile b/package/libs/libjson-c/Makefile
--- a/package/libs/libjson-c/Makefile
+++ b/package/libs/libjson-c/Makefile
@@ -20,6 +20,7 @@ PKG_LICENSE_FILES:=COPYING
 
 PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=0
 
 PKG_MAINTAINER:=Felix Fietkau <nbd@openwrt.org>
 
